config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -236,13 +236,6 @@ presubmits:
         - '--test-args=--container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/home/containerd/usr/local/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd\"]}"'
         - --image-config-file=/go/src/k8s.io/test-infra/jobs/e2e_node/containerd/containerd-main/image-config.yaml
         - --image-config-dir=/home/prow
-    resources:
-      requests:
-        cpu: 4
-        memory: 6Gi
-      limits:
-        cpu: 4
-        memory: 6Gi
```

Comment:
Please fix indentation rather than deleteing.

Reply:
it is added above as well in the on L-217  ... this was extra